viadot/flows/outlook_to_adls.py

```diff
@@ -47,13 +47,15 @@ def __init__(
             adls_sp_credentials_secret (str, optional): The name of the Azure Key Vault secret containing a dictionary with
             ACCOUNT_NAME and Service Principal credentials (TENANT_ID, CLIENT_ID, CLIENT_SECRET) for the Azure Data Lake. Defaults to None.
             limit (int, optional): Number of fetched top messages. Defaults to 10000.
+            timeout (int, optional): The amount of time (in seconds) to wait while running this task before a timeout occurs. Defaults to 1200.
```

Comment:
This is a flow not task, I think you meant sth like "The amount of time (in seconds) to wait for downloading Outlook data before a timeout occurs. Defaults to 1200".